web/beacon-app/src/features/onboarding/components/steps/preference/index.tsx

```diff
@@ -51,7 +51,8 @@ const UserPreferenceStep = () => {
               label: item,
               value: item,
             })),
-            profession_segment: profile.profession_segment,
+            profession_segment:
+              profile?.profession_segment !== 'Unspecified' ? profile?.profession_segment : '',
```

Comment:
This will ensure that the empty string is set as default value when the BE sends `Unspecified`